### DISCUSSION TITLE
conditionally render handles

### DISCUSSION THREAD

**Discussion by navaniharsh31** [xyflow/xyflow#2873] [Q&A] [answered]
While conditionally rendering handles, connection lines and edges are not being rendered. 
Is it happening because of conditionally rendering it or might be some other issue?

**Comment by bcakmakoglu** [comment, marked as answer]
[Check this part of the docs regarding dynamic handles.](https://reactflow.dev/docs/api/hooks/use-update-node-internals/#usage)

**Reply by navaniharsh31** [reply]
thanks!
